### PR TITLE
Remove mergify

### PR DESCRIPTION
**Pull request by jpkrohling**
GitHub has added a few features, such as "merge on green", reducing the appeal of Mergify. 

﻿Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Review by yurishkuro — DISMISSED** [review]

**Comment by yurishkuro** [comment]
Can gh also  dismiss state reviews?

**Comment by jpkrohling** [comment]
Yes, here:

image


I went ahead and enabled it preemptively. If you think we shouldn't, disable it back. 

cc @jaegertracing/jaeger-maintainers

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**
